### PR TITLE
Add a Code of Conduct : issue #19

### PR DESCRIPTION
**Pull request by tarunjain1st**
file created & link added

### CODE REVIEW

File: project_2_image_captioning_project/DEVELOPERS.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ When done with the Setup and Installation process it time to test our project an
 
 ## Contribution
 If you’re interested in the project, feel free to open an issue, create a PR, or just come say hi and tell us more about yourself.
+
+`Please Read : ` [Code of conduct](code-of-conduct.md)
```
Comment (vidhi-mody) [inline project_2_image_captioning_project/DEVELOPERS.md]: Add this in the README.
Don't wrap the Please read in ``

File: project_2_image_captioning_project/code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Code of Conduct
+
+This code of conduct outlines our expectations for participants within the [Student Code-In](https://scodein.tech) community, as well as steps to reporting unacceptable behavior. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored. Anyone who violates this code of conduct may be banned from the community.
```
Comment (vidhi-mody) [inline project_2_image_captioning_project/code-of-conduct.md]: Please remove the StudentCodeIn part. These are general guidelines for all contributors.

**Review by vidhi-mody — CHANGES_REQUESTED** [review]

File: project_2_image_captioning_project/DEVELOPERS.md (hunk 1 above)
Comment (tarunjain1st) [reply project_2_image_captioning_project/DEVELOPERS.md]: done

File: project_2_image_captioning_project/code-of-conduct.md (hunk 2 above)
Comment (tarunjain1st) [reply project_2_image_captioning_project/code-of-conduct.md]: done

**Comment by vidhi-mody** [comment]
@tarunjain1st are you making a new PR?

**Pull request closed without merging**